crates/sound/src/vhu_sound.rs

```diff
@@ -113,14 +124,14 @@ impl VhostUserSoundThread {
                     // new requests on the queue.
                     loop {
                         vring.disable_notification().unwrap();
-                        self.process_control(vring, stream_info)?;
+                        self.process_control(vring, &audio_backend, stream_info)?;
```

Comment:
What about passing it when we create `VhostUserSoundThread` ?

Reply:
Do you mean to define the audio_backend at `struct VhostUserSoundThread{}`?

Reply:
yep, if that can't change over the life of the thread, maybe it's better to pass it on just once at creation. What do you think?

Reply:
This makes more sense, and the AudioBackend should have interior mutability only, not using &mut references anywhere.

Reply:
Yes, I can do that, thanks. @epilys I am not familiar with mutability, so could you explain why AudioBackend should have "internal mutability" only? Thanks.